src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -12,8 +12,8 @@
         <tr>
           <td i18n
               class="bold">Storage Capacity</td>
-          <td><span i18n>Number of devices: {{ totalDevices }}. Raw capacity:
-            {{ totalCapacity | dimlessBinary }}.</span></td>
+          <td><span i18n>Number of devices: {{ totalDevices }}<br>Raw capacity:
```

Comment:
For line breaks it's better to avoid `<br />` and use CSS for achiebing it. See:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns

Reply:
Done. Thanks @alfonsomthd